packages/wrangler/src/worker.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "worker:*" {
```

Comment:
Declaring the type for `worker:` prefix module imports. We added an esbuild plugin to find these worker files in `packages/wrangler/templates/*`